libs/shared/ui-components/src/SocialCard/types.ts

```diff
@@ -0,0 +1,10 @@
+import { DomainVariant } from '@quansight/shared/types';
+
+export type TSocailCardProps = {
```

Comment:
Typo here: `TSocialCardProps` -- will need to be propagated to all imports and points of use